techniques/system/common/1.0/metadata.xml

```diff
@@ -76,7 +76,6 @@
   </FILES>
 
   <SYSTEMVARS>
-    <NAME>AUTHORIZED_NETWORKS</NAME>
```

Comment:
it's still used in common technique as a system var, you need to call it `ALLOWED_NETWORKS` here